packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -71,20 +74,41 @@
             />
           </div>
         </li>
+        <li>
+          
+          <oc-button id="oc-topbar-notifications" appearance="raw" @click="openNotificationsModal">
+            <oc-icon name="notification" fill-type="line" class="oc-p-xs"/>
+              <span v-text="$gettext('Notification Settings')"/>
+          </oc-button>
+        </li>
       </oc-list>
     </oc-drop>
+    
   </nav>
+    <div>
+      <notifications-modal
+          v-if="notificationsModalOpen"
+          v-model:initialNotifyEnabled="checkboxValue"
+          @cancel="closeNotificationsModal"
+          @confirm="updateNotificationPreference"
+      ></notifications-modal>
```

Comment:
Vue components can be self-closing.

```suggestion
      />
```